DataFormats/Detectors/ITSMFT/common/src/TopologyDictionary.cxx

```diff
@@ -184,5 +185,17 @@ Point3D<float> TopologyDictionary::getClusterCoordinates(const CompCluster& cl)
   return locCl;
 }
 
+Point3D<float>
```

Comment:
``cl`` is used only to get the coodinates of the reference pixel. Why not using them directly insteaf of the whole compact cluster?

Reply:
I would say, it's a question taste.  With your variant, you would always need to type more arguments.
Also, the name of the function is "getClusterCoordinates", which calls for an argument to be a cluster.
I would keep it as it is.

Reply:
OK, I agree